examples/guided.py

```diff
@@ -319,7 +319,7 @@ def perform_installation(mountpoint):
 			archinstall.use_mirrors(archinstall.arguments['mirror-region']) # Set the mirrors for the live medium
 		if installation.minimal_installation():
 			installation.set_hostname(archinstall.arguments['hostname'])
-			if archinstall.arguments['mirror-region'].get("mirrors",{})!= None:
+			if archinstall.arguments['mirror-region'].get("mirrors", {}) is not None:
```

Comment:
Issue outside of this PR:
But I believe this check is incorrect. `{}` will trigger as `True` against `is not None` and carry on attempting to set an empty mirror list? Which is not what we want. Perhaps omit anything after and just do `if get("mirrors", {})` to make sure it evaluates `True` only if mirrors were actually given? Or do `.get("mirrors", None) is not None`.

Reply:
Changed to your suggestion.